tests/test_lllaplace.py

```diff
@@ -27,6 +30,24 @@ def model():
     return model
 
 
+@pytest.fixture
+def model_with_reduction():
+    class Model(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.fc1 = nn.Linear(3, 20)
+            self.fc2 = nn.Linear(20, 2)
+            self.output_size = 2
+
+        def forward(self, x):
+            x = self.fc1(x)
+            x = nn.functional.relu(x)
+            x = x.mean(1)
```

Comment:
What's the reason this model only applies the mean reduction and doesn't have the option for `PICK_FIRST` or `PICK_LAST`?

Reply:
The options in `Laplace` are completely detached to the one made by the model (`Laplace` has no way knowing that). The user has to take care of this themselves. It is noted both here
https://github.com/aleximmer/Laplace/blob/ff139fe936148fc5e7ace56b26917b453d75e0bc/laplace/lllaplace.py#L61-L63
and here
https://github.com/aleximmer/Laplace/blob/ff139fe936148fc5e7ace56b26917b453d75e0bc/examples/huggingface_example.py#L106-L108


But in any case, for test purposes, setting the mock model to have just `mean` is sufficient.